transcom/mymove
# B-18065

## [B-18065](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A870632)

## Summary

Integration review found [here](https://github.com/transcom/mymove/pull/12250).

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login into Milmove customer
2. on board PPM and submit
3. Login into Milmove office and review move
4. Approve PPM and "move details"
5. Login into Milmove customer and start uploading documentations
6. Add weights, pro-gear, expenses ..etc..
7. Log in to Milmove Office to approve customer payment
8. Verify payment packet download is available for Office user (see screenshot)
9. Log in to Milmove customer and verify payment packet download is available 

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
